Agrega tarea 1 de la clase 10

YamiRampello opened the pull request:
A la tarea 1 de la clase 6 se agregan estilos con bootstrap. Se trabajó sobre la solución de Fabricio Sodano explicada en clase.

YamiRampello committed:
Agrega tarea 1 de la clase 10

A la tarea 1 de la clase 6 se agregan estilos con bootstrap. Se trabajó sobre la solución de Fabricio Sodano explicada en clase.

YamiRampello committed:
Agrega tarea 2 de la clase 10

A la tarea 2 de la clase 6 se agregan estilos con bootstrap.

YamiRampello commented:
![clase10-tarea2](https://github.com/YamiRampello/r-argentina-programa-introduccion-a-js-tareas/assets/99457830/f6040bf6-e120-48d2-9b25-88142d19b7b8)

YamiRampello commented:
Clase 6 - tarea 1
![clase10-tarea1](https://github.com/YamiRampello/r-argentina-programa-introduccion-a-js-tareas/assets/99457830/8f2c6aea-62f5-4553-97ba-62e26743f57b)

GonzaloAlt commented on clase-10/clase-6-tarea-1-FS.js:
```diff
@@ -0,0 +1,209 @@
+const $form = document.querySelector('#calculador-edades');
+
+document.querySelector('#siguiente-paso').onclick = function (event) {
+  const identificaError = 'cantidadIntegrantes';
```
Tal vez identificadorError como nombre de variable?

GonzaloAlt commented on clase-10/clase-6-tarea-1-FS.js:
```diff
@@ -0,0 +1,209 @@
+  manejarErrores(errores, identificaError);
+  borrarIntegrantesAnteriores();
+
+  if (errorCantidadIntegrantes === '') {
+    crearIntegrantes(cantidadIntegrantes);
+  } else {
+    ('');
+  }
```
Esto podría ser `if(!errorCantidadIntegrantes) crearIntegrantes(cantidadIntegrantes);`
https://developer.mozilla.org/en-US/docs/Glossary/Falsy

GonzaloAlt commented on clase-10/clase-6-tarea-1-FS.js:
```diff
@@ -0,0 +1,209 @@
+    const error = errores[key];
+
+    if (identificaError === 'cantidadIntegrantes') {
+      if (error === '') {
+        $form[key].className = '';
+      } else {
+        $form[key].className = 'error';
+        const $error = document.createElement('li');
+        $error.innerText = error;
+        $errores.appendChild($error);
```
Esto podría ir dentro de una función. Y acordate `if (!error)`.

GonzaloAlt commented on clase-10/clase-6-tarea-1-FS.js:
```diff
@@ -0,0 +1,209 @@
+        $errores.appendChild($error);
+      }
+    } else {
+      for (let i = 0; i < error.length; i++) {
+        if (error[i] === '') {
+          $form[key][i].className = '';
+        } else {
+          $form[key][i].className = 'error';
+          const $error = document.createElement('li');
+          $error.innerText = error[i];
+          $errores.appendChild($error);
+        }
```
Esto también podría ir dentro de una función. Y acá también `if(!error[i])`

GonzaloAlt commented on clase-10/clase-6-tarea-2.js:
```diff
@@ -0,0 +1,229 @@
+  let contadorErorr = 0;
+
+  for (let i = 0; i < numeros.length; i++) {
+    if (validarSalario(numeros[i]) !== '') {
+      contadorErorr++;
+    } else {
+      ('');
+    }
```
`if (!validarSalario(numeros[i]))`

GonzaloAlt commented on clase-10/clase-6-tarea-2.js:
```diff
@@ -0,0 +1,229 @@
+    document.querySelector('#mayor-salario').value = maximoSalarios;
+    document.querySelector('#promedio-salario-mensual').value =
+      mediaSalarioMensual;
+  } else {
+    ('');
+  }
```
No hace falta el `else`, se puede hacer `if(condición){} `sin necesidad de un `else`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else

GonzaloAlt commented on clase-10/clase-6-tarea-2.js:
```diff
@@ -0,0 +1,229 @@
+    const error = errores[key];
+
+    if (identificaError === 'integrantes') {
+      if (error === '') {
```
`if(!error)`

YamiRampello committed:
Agrega modificaciones según comentarios

En las correcciones se marcó que modifique los condicionales y el nombre de una constante
